exp-player/addon/components/exp-overview/component.js

```diff
@@ -107,13 +107,16 @@ export default ExpFrameBaseComponent.extend(Validations, {
     type: 'exp-overview',
     layout: layout,
     questions: questions,
-    responses: Ember.computed('questions', function() {
-        var questions = this.get('questions');
-        var responses = {};
-        for (var i=0; i < questions.length; i++) {
-            responses[i] = questions[i].value;
-        }
-        return responses;
+    showOptional: Ember.computed('questions.7.value', function() {
```

Comment:
Oh, I think I was confusing lists with objects -- `questions.@each.value` would also work, but that would watch all values instead of just `question[7]`